### PR TITLE
Update chapter 3, 4, 5, and 6 up to 6.3.

### PR DESCRIPTION
**Pull request by neilt**
Updates include corrections and content for use with LedgerSMB version 1.9.
This updates puts in place basic content and still needs to be wordsmithed and formatted.
Some placeholder images were added or changed that will need to be updated later.

### CODE REVIEW

File: part-getting-started.tex [hunk 1]
```diff
@@ -276,32 +288,99 @@ \subsection{Login screen}
 \item[Company] The name of the database created; Jack uses \texttt{example\_inc}
 \end{description}
 
+After entering all of the information and taping the \texttt{Login} button Jack will see an alert letting
```
Comment (ehuelsmann) [inline part-getting-started.tex:291]: `taping`? or `tapping`?

File: part-overview.tex [hunk 2]
```diff
@@ -133,7 +133,7 @@ \section{Supported functionality}
 \item Application administration
 \end{itemize}
 
-These add-ons can be installed:
+These \glspl{add-on}  can be installed:
```
Comment (ehuelsmann) [inline part-overview.tex:136]: Ah. You can't know this yet, but budgetting has been integrated into the application, so it's no longer an add-on. I'm not aware of "enhanced AR/ AP" being an option these days (and although I wrote it, I don't remember what it was actually about...)

File: part-getting-started.tex (hunk 1 above)
Comment (neilt) [reply part-getting-started.tex:291]: Done in next update.

File: part-overview.tex (hunk 2 above)
Comment (neilt) [reply part-overview.tex:136]: Removed both in the next update.
Thanks

**Pull request merged**